loading-button-android/build.gradle

```diff
@@ -5,7 +5,7 @@ apply plugin: 'com.github.dcendents.android-maven'
 apply plugin: 'org.jmailen.kotlinter'
 
 //Bintray depends on this global variable to set the library version!
-version = "2.0.2"
+version = "3.0.0"
```

Comment:
Version shouldn't be 3.0.0. There's no breaking changes since 2.0.0. The insertion of androidx was the breaking change in 1.14.0